src/main/java/training_day4/HW/Car.java

```diff
@@ -0,0 +1,12 @@
+package training_day4.HW;
+
+public class Car extends Vehicle {
+
```

Comment:
Keep only one empty line after the beginning of the class. Then write all other data.